tests/AbstractRepositoryTest.php

```diff
@@ -718,4 +765,78 @@ public function testFindExpectsStringPath()
 
         $repo->find(new \stdClass());
     }
+
+    public function testAutoresolveLinks()
```

Comment:
You should split this test into separate tests and give them descriptive names, then you can spare the comments inside. As it is right now, a failing test does not say anything about the actual problem ("autoresolve links fails").